plugins/SteamConnect/class.steamconnect.plugin.php

```diff
@@ -121,18 +165,29 @@ public function openIDPlugin_afterConnectData_handler($sender, $args) {
         }
     }
 
+    /**
+     * Get steam id
+     *
+     * @param $openID
+     * @return mixed
```

Comment:
"mixed" would typically imply you can't be sure what's returned, but it looks like there are only a couple of possibilities here.